### PR TITLE
Feature/e commerce protos Oriol Bover

### PR DESCRIPTION
**Pull request by boversauros**

### CODE REVIEW

File: staff/oriol-bover/e-commerce-protos/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (martimalek) [inline staff/oriol-bover/e-commerce-protos/index.html]: Maybe you could update your title, something like "e-commerce" should do.

File: staff/oriol-bover/e-commerce-protos/scripts/cart.demo.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+var cart = new Cart;
+
+// - PRODUCTS
```
Comment (martimalek) [inline staff/oriol-bover/e-commerce-protos/scripts/cart.demo.js:3]: Great comments!

File: staff/oriol-bover/e-commerce-protos/index.html (hunk 1 above)
Comment (boversauros) [reply staff/oriol-bover/e-commerce-protos/index.html]: Thanks for the observation!, Done

File: staff/oriol-bover/e-commerce-protos/scripts/cart.demo.js (hunk 2 above)
Comment (boversauros) [reply staff/oriol-bover/e-commerce-protos/scripts/cart.demo.js:3]: Thank you!

**Pull request merged**